packages/core/core/src/plugins.ts

```diff
@@ -31,6 +32,8 @@ export class Plugins {
 
   version: string;
 
+  client: string;
```

Comment:
It's also a bit odd that all plugins will have a client property when the plugin may have nothing to do withclient.

Reply:
see above